admin/load.php

```diff
@@ -336,3 +336,88 @@ function perflab_get_module_data( $module_file ) {
 
 	return $module_data;
 }
+
+/**
+ * Initialise Admin Pointer
+ *
+ * Handles the bootstrapping of the admin pointer.
+ * Mainly jQuery code that is self-initialising.
+ *
+ * @since 1.0.0
+ * @return void
+ */
+function perflab_admin_pointer() {
+
+	wp_enqueue_style( 'wp-pointer' );
+	wp_enqueue_script( 'wp-pointer', false, array( 'jquery' ) );
+
+	$current_user = get_current_user_id();
+	$meta_key     = 'my-pointer-slug-dismissed';
```

Comment:
I think this one might need a tweak. Something along the lines of:

```suggestion
	$meta_key     = 'performance-initial-pointer';
```

Reply:
I think it would be better to use WordPress core's intended storage for this instead of introducing a new meta key altogether. We can use code like this:
```php
$dismissed = explode( ',', (string) get_user_meta( get_current_user_id(), 'dismissed_wp_pointers', true ) );
```

What you have here should then rather be the "pointer ID" used to put into that array if the user dismisses it.

Reply:
Ran with @felixarntz suggestion here.